### PR TITLE
New logger folder

### PR DESCRIPTION
**Pull request by dalderliesten**
Updated the location of the logging folder and files so that they are in a better location.  Also changed the gitignore to no longer add the logging files and other items in the userworks folder.  Also fixed some styling in the constants.

<feature, organization

### CODE REVIEW

File: src/main/java/nl/tudelft/scrumbledore/Constants.java [hunk 1]
```diff
@@ -118,15 +117,18 @@ private Constants() {
   // References to images utilized for the game display within the GUI.
   public static final String RESOURCES_DIR = "src/main/resources/";
   public static final String SPRITES_DIR = "images/sprites/";
+  public static final String USERWORKS_DIR = "src/main/userworks";
```
Comment (jessetilro) [inline src/main/java/nl/tudelft/scrumbledore/Constants.java]: This path should refer to a directory outside of the src directory.

Comment (dalderliesten) [inline src/main/java/nl/tudelft/scrumbledore/Constants.java]: Understood, fixing it now to be out of the src directory.

<organization

**Comment by dalderliesten** [comment]
Fixed the appdata/logger location as we discussed during the meeting.

File: src/main/java/nl/tudelft/scrumbledore/Scrumbledore.java [hunk 2]
```diff
@@ -26,4 +33,25 @@ public static void main(String[] args) {
     GUI.launch(GUI.class);
   }
 
+  /**
+   * Creates the appData folder, which is used by multiple classes for issues such as logging and
+   * keybinding saving.
+   */
+  private static void makeAppData() {
```
Comment (NielsWarnars) [inline src/main/java/nl/tudelft/scrumbledore/Scrumbledore.java]: I would rename this method to makeAppDataDir

File: src/main/java/nl/tudelft/scrumbledore/Scrumbledore.java [hunk 3]
```diff
@@ -26,4 +33,25 @@ public static void main(String[] args) {
...
+    File appDataDir = new File(Constants.APPDATA_DIR);
+
+    try {
+      if (!appDataDir.exists()) {
+        boolean result = appDataDir.mkdir();
+
+        // Throw an IOException if the logging directory could not be made.
```
Comment (NielsWarnars) [inline src/main/java/nl/tudelft/scrumbledore/Scrumbledore.java]: Comment can be changed so that it describes the why instead of the what

**Pull request merged**